diff --git a/node_modules/react-native-purchases/ios/RNPurchases.m b/node_modules/react-native-purchases/ios/RNPurchases.m
index 9179f10..cd16342 100644
--- a/node_modules/react-native-purchases/ios/RNPurchases.m
+++ b/node_modules/react-native-purchases/ios/RNPurchases.m
@@ -475,9 +475,6 @@ static void logUnavailablePresentCodeRedemptionSheet() {
     [RCCommonFunctionality setCreative:creative.mappingNSNullToNil];
 }
 
-RCT_EXPORT_METHOD(setAppsFlyerConversionData:(NSDictionary *)data) {
-    [RCCommonFunctionality setAppsFlyerConversionData:data.mappingNSNullToNil];
-}
 
 RCT_EXPORT_METHOD(overridePreferredLocale:(nullable NSString *)locale) {
     [RCCommonFunctionality overridePreferredLocale:locale.mappingNSNullToNil];
